virtio-devices/src/vdpa.rs

```diff
@@ -351,6 +355,7 @@ impl<M: GuestAddressSpace + Sync + Send> ExternalDmaMapping for VdpaDmaMapping<M
     }
 
     fn unmap(&self, iova: u64, size: u64) -> std::result::Result<(), std::io::Error> {
+        info!("DMA unmap iova 0x{:x} size 0x{:x}", iova, size);
```

Comment:
How often does this get reported? Are there matching ones for other device types that sit behind the vIOMMU?

Reply:
It gets reported quite a lot. What do you have in mind here?

Reply:
Then it should be `debug!`

Reply:
Alright!